expirable_cache.go

```
@@ -45,6 +54,7 @@ func NewExpirableCache(opts ...Option) (*ExpirableCache, error) {
 				size := s.Size()
 				atomic.AddInt64(&res.currentSize, -1*int64(size))
 			}
+			_ = res.eventBus.Publish(res.id, key)
```

will be nice to have a comment explaining why we ignored the error.